tests/Integration/CompletorTestCase.php

```diff
@@ -47,6 +47,10 @@ protected function assertComplete(string $source, array $expected)
 
         foreach ($expected as $index => $expectedSuggestion) {
             $this->assertArraySubset($expectedSuggestion, $actual[$index]);
+
+            if (!isset($expectedSuggestion['class_import']) || $expectedSuggestion['class_import'] === null) {
+                $this->assertNull($actual[$index]['class_import']);
+            }
```

Comment:
hmm, is this not covered by array subset above?

Reply:
yes,  it's useless. I started with the wrong assumption the 'class_import' key was absent instead of present and associated to ``null``